Formula/ponyc.rb

```diff
@@ -1,10 +1,9 @@
 class Ponyc < Formula
   desc "Object-oriented, actor-model, capabilities-secure programming language"
   homepage "http://www.ponylang.org"
-  url "https://github.com/ponylang/ponyc/archive/0.2.1.tar.gz"
-  sha256 "cb8d6830565ab6b47ecef07dc1243029cef962df7ff926140022abb69d1e554e"
-  revision 1
-  head "https://github.com/ponylang/ponyc.git"
+  url "https://github.com/ponylang/ponyc.git",
+    :revision => "aafebac938273d4786c02dfb2ba9ec7a164675e7"
+  version "0.2.2-candidate"
```

Comment:
That was the latest commit at the time I was writing that update. Why does `-candidate` would compare more and `-alpha` not? I don't know how version sorting works, but simple test yields:
```ruby
irb(main):001:0> a = ["0.2.2-candidate", "0.2.2", "0.2.1", "0.2.2-alpha"]
=> ["0.2.2-candidate", "0.2.2", "0.2.1", "0.2.2-alpha"]
irb(main):002:0> a.sort
=> ["0.2.1", "0.2.2", "0.2.2-alpha", "0.2.2-candidate"]
```